ideascube/monitoring/forms.py

```diff
@@ -110,24 +113,28 @@ class LoanForm(forms.ModelForm):
 
     def clean_specimen(self):
         barcode = self.cleaned_data['specimen']
-        if Loan.objects.due().filter(specimen__barcode=barcode).exists():
+        if (Loan.objects.due().filter(specimen__barcode=barcode).exists()
+         or Loan.objects.due().filter(specimen__serial=barcode).exists()):
```

Comment:
WARNING: A long digression not really related to this PR

I would add one or two space between the `if` and the `(` to keep `Loan...` aligned and keep an indentation between `if` and `and` :

```python
if  (A.very.long.line.test()
 and D.very.long.line.test()
 and (B.very.long.line.test()
   or C.very.long.line.test())):
    foo()
```

I agree that is not perfect but I'm not convinced neither by the pep8 solution :

```python
if (A.very.long.line.test()
    and D.very.long.line.test()
    and (B.very.long.line.test()
         or C.very.long.line.test())):
    foo())
```

In this context, I would have prefer the prefix notation, but that's not python :
```python
if and(A.very.long.line.test(),
       D.very.long.line.test(),
       or(B.very.long.line.test(),
          C.very.long.line.test())):
    foo()
```

or a python version (but less readable) :
```python
if all([A.very.long.line.test(),
        D.very.long.line.test(),
        any([B.very.long.line.test(),
             C.very.long.line.test()])]):
    print('a')
```

Maybe the solution is to use intermediate variables and write the `if` condition in one line...
... what I will do :)